ex04.ex

```diff
@@ -75,8 +78,10 @@ defmodule Ex04 do
   return value will be the thing you have to manipulate.
   """
 
-  def even_odd . . . "your code"
-
+  def even_odd([]), do: nil
```

Comment:
# +1

So that's clever: reversing the input list means you don't have to reverse both result lists.

That saved you from getting a deduction for line 82, which has way too much going on...